src/components/dialog/dialog.scss

```diff
@@ -80,7 +80,7 @@ md-dialog {
     }
   }
 
-  .md-actions, md-dialog-actions {
```

Comment:
We had them since pre v1.0.0 that's why I intentionally removed them for now. The target is to remove the deprecation completely (two major versions~)